docs/reference/production.md

```diff
@@ -0,0 +1,53 @@
+# Production deployment
+
+The currently supported configuration for production deployments consist of a single
+physical server running the entire core network. This includes both the control plane
+and the user plane, and the observability stack.
+
+All the components of the gNodeB are not covered in this deployment and should be run
+externally.
+
```

Comment:
We should probably link to the how to guide somewhere here. 

Ex:

```
see also: [how-to deploy a production deployment](../how-to/production.md)
```

Reply:
I added the link.